app/serializers/export/contact_serializer.rb

```diff
@@ -1,12 +1,28 @@
 module Export
   class ContactSerializer < ActiveModel::Serializer
     attributes :sharing,
-               :receiving,
+               :following,
```

Comment:
Could you please decribe in more details what is the issue with privacy when someone sets `receiving` to true before import? I didn't quite get it.